backend/gamify/models.py

```diff
@@ -1,19 +1,10 @@
-import hashlib
-import time
-
 from django.db import models
 
 
 class Team(models.Model):
     name = models.CharField(max_length=100, blank=False, null=False)
     join_code = models.CharField(max_length=100, blank=False, null=False)
 
-    # Factory method b/c messing with constructor might break other things
-    def create(cls, name):
```

Comment:
I couldn't figure out why join code wasn't being assigned within this function, so I decided to stop wasting time on debugging and just remove it